Misc/NEWS.d/next/Core and Builtins/2022-07-29-09-52-22.gh-issue-95380.p8S_K0.rst

```diff
@@ -0,0 +1 @@
+Remove max path size limit of 1024 for fcntl function.
```

Comment:
I believe that CPython users should know what functions were affected. In addition, a patch is neither primitive (one-liner) nor a follow-up so no need  to keep it anonymous.

```suggestion
Remove max path size limit of 1024 for :func:`fcntl.fcntl` function. Patch by
Grégory Starck.
```